src/locales/en.json

```diff
@@ -68,5 +68,12 @@
   "Variance reason": "Variance reason",
   "Variance updated successfully": "Variance updated successfully",
   "View": "View",
-  "You do not have permission to access this page": "You do not have permission to access this page"
+  "You do not have permission to access this page": "You do not have permission to access this page",
```

Comment:
Arrange the changes in locale files alphabetically.

Reply:
done